### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by Flouse**
add Network Status etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 * [Mainnet_v0](./mainnet_v0)
 
-## Documentation
+## Network Status
 
-* [Getting Started with Godwoken](https://startwithnervos.com/godwoken)
-* [How to test your Solidity contracts on Godwoken v1 using Hardhat](https://github.com/nervosnetwork/godwoken-tests)
-* https://docs.godwoken.io
-* [Godwoken CHANGELOG](https://github.com/nervosnetwork/godwoken/blob/develop/CHANGELOG.md)
+- https://godwoken.statuspage.io
```
Comment (Flouse) [inline README.md:9]:
```suggestion
- https://godwokenstatus.statuspage.io/
```

Comment (Simon-Tl) [reply README.md:9]: No longer use https://godwoken.statuspage.io ？

Comment (Flouse) [reply README.md:9]: I think it is ready to use https://godwoken.statuspage.io

Comment (Simon-Tl) [reply README.md:9]: > https://godwoken.statuspage.io

yes，Now it's working properly

**Pull request merged**